lib/fizzy/capi.cpp

```diff
@@ -473,7 +475,7 @@ bool fizzy_find_exported_function_index(
 }
 
 bool fizzy_find_exported_function(
-    FizzyInstance* instance, const char* name, FizzyExternalFunction* out_function)
+    FizzyInstance* instance, const char* name, FizzyExternalFunction* out_function) noexcept
```

Comment:
`wrap` call below can throw `bad_alloc`, we should add `catch(...) { return false; } `  for now, later adding rich error reporting to it or getting rid of dynamic allocation somehow.

Reply:
I wonder why clang-tidy didn't complain about this...

Reply:
Added.